### PR TITLE
Add draft configuration for calculator

### LINKED ISSUE #51: Add draft configuration file and icon to calculator

### PR DESCRIPTION
**Pull request by matteodelabre**
Fixes #51.

### CODE REVIEW

**Comment by Eeems** [comment]
You are including both svg and png?

**Comment by matteodelabre** [comment]
Would it be better to generate the PNG on the fly? That would require adding `inkscape` to the build environment, but it seems feasible.

**Comment by Eeems** [comment]
> 
> 
> Would it be better to generate the PNG on the fly? That would require adding `inkscape` to the build environment, but it seems feasible.

Probably, otherwise it's not clear what the svg is being used for.

**Comment by matteodelabre** [comment]
That would be so that others can iterate on the icon if needed.

**Comment by LinusCDE** [comment]
Please also ensure though that builds are repeatable. Converting svg to png with inkscape also seems overkill.

They have indeed some custom metadata in their svg files which can make converting/rendering faulty with other software. But then this is akin to a IDEA-Specific project file and one should usually export the plain svg file in my opinion, which should convert fine with any svg to pdf converting utility.

**Comment by matteodelabre** [comment]
> They have indeed some custom metadata in their svg files which can make converting/rendering faulty with other software. But then this is akin to a IDEA-Specific project file and one should usually export the plain svg file in my opinion, which should convert fine with any svg to pdf converting utility.

Agreed. This PR contains the plain SVG file.

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
               uses: actions/checkout@v2
             - name: Install missing tools
               run: |
-                sudo apt-get install bsdtar tree
+                sudo apt-get install bsdtar inkscape tree
```
Comment (Eeems) [inline .github/workflows/testing.yml]: Do we still need this change?

Comment (matteodelabre) [reply .github/workflows/testing.yml]: Good catch!

**Review by Eeems — APPROVED** [review]

**Pull request merged**